internal/organizations/org.go

```diff
@@ -42,10 +42,10 @@ type Qset struct {
 // BuildGraph makes a graph from the Gleaner config file source
 // load this to a /sources bucket (change this to sources naming convention?)
 func BuildGraph(mc *minio.Client, v1 *viper.Viper) error {
-	var (
-		buf    bytes.Buffer
-		logger = log.New(&buf, "logger: ", log.Lshortfile)
-	)
+	//var (
```

Comment:
It looks like you might be able to delete these lines, and their subsequent friends - i guess `buf` just isn't ever used?